src/addons/Portal/Portal.js

```diff
@@ -363,6 +363,14 @@ class Portal extends Component {
     )
 
     this.portal = this.node.firstElementChild
+    // don't take focus away from portals that close on blur
+    if (!closeOnTriggerBlur) {
+      this.triggerNode = document.activeElement
+      this.portal.setAttribute('tabindex', '-1')
+      this.portal.style.outline = 'none'
+      // wait a tick for things like popups which need to calculate where the popup shows up
+      setTimeout(() => this.portal && this.portal.focus())
```

Comment:
Oh, I see, thanks for the clarification.  From my initial pass I assumed this was called in a `render()` method itself.  -1 for assumptions :)